lib/stencil-download.utils.js

```diff
@@ -0,0 +1,137 @@
+const request = require("request");
+const yauzl = require('yauzl');
+const fs = require('fs');
+const tmp = require('tmp');
+const path = require('path');
+const utils = {};
+
+module.exports = utils;
+
+utils.downloadThemeFiles = (options, callback) => {
+    tmp.file(function _tempFileCreated(err, tempThemePath, fd, cleanupCallback) {
+        if (err) {
+            callback(err);
+        }
+
+        (
+            new Promise(
+                (resolve, reject) =>
+                    request(options.downloadUrl)
+                        .pipe(fs.createWriteStream(tempThemePath))
+                        .on('finish', () => resolve(tempThemePath))
+                        .on('error', reject),
+            )
+        )
```

Comment:
The indentation looks off here on these 2 closing `)`

Reply:
@junedkazi - I think the indentation here is okay.